docs/servers/add-azure-server.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Add a Server to Azure
+intro: Devopness allows you to provision a server on Azure Resource Manager (RM) and manage it through Devopness.
+links:
+    overview:
+    quickstart:
+    previous: /docs/servers/add-server
+    next: /docs/servers/check-server-provision-logs
+    guides:
+    related:
+    featured:
+---
+
+> If you don't have an Azure credential, please follow the guide {% mentionPost "/docs/credentials/add-azure-credential" %}
+
+1. On Devopness, navigate to a project then select an environment
+1. If there's no other server in the selected environment, just click `ADD SERVER`
+1. If a list of resource cards is displayed, find the `Servers` card and click View on it
```

Comment:
@Diegiwg these steps do not reflect the current web app UX.

In the past, when accessing the environment modules page on an environment without a server, it would show a page guiding the user to add a server.

Could you update these steps to reflect the current UX?

```suggestion
1. Find the `Servers` card
1. Click `View` in the `Servers` card to see a list of existing `Servers`
```

Reply:
@Diegiwg it seems the same problem can be found on the other add-{aws, gcp, existing, digitalocean}-server

Could you apply the same changes to the other articles?

If you choose to open this as a separate PR, you can tag me to review it

![image](https://github.com/user-attachments/assets/a973f5df-0db3-4a45-8745-bf65b55602b8)